### PR TITLE
Add info about XZG Firmware

### PR DESCRIPTION
**Pull request by xyzroe**

### CODE REVIEW

File: docs/guide/adapters/zstack.md [hunk 1]
```diff
@@ -22,7 +22,9 @@ Adapters based on CC1352 or CC2652 chips can be flashed by putting them in the b
   - [TubesZB TI CC2652 FW Flasher](https://github.com/tube0013/tubeszb_addons)
   - [ZigStar TI CC2652 FW Flasher](https://github.com/mercenaruss/zigstar_addons) (fork of TubesZB with added features)
 
-- Some Ethernet adapters support flashing Zigbee firmware over their own web-interface. In this case you do not need any external software and hardware. Just go to the webinterface and press "Update Zigbee firmware". Please refer to the manual of your particular Zigbee adapter for this functionality.
+- Some Ethernet adapters support flashing Zigbee firmware over their own web-interface. In this case you do not need any external software and hardware. Just go to the webinterface and press "Update Zigbee firmware". Please refer to the manual of your particular Zigbee adapter for this functionality or just use universal [XZG Firmware](https://github.com/xyzroe/XZG) that fits any CC2652* based gateway
```
Comment (Koenkk) [inline docs/guide/adapters/zstack.md:25]: This is adapter specific and should move to the adapter section instead.

**Comment by xyzroe** [comment]
It makes it possible to easily update the cc2652 firmware on any gateway with esp32. But you are the author, so it's up to you to decide. Should I move it? Or just close?

**Comment by Koenkk** [comment]
But only on gateways running this particular firmware right?

**Comment by xyzroe** [comment]
Right

**Comment by Koenkk** [comment]
Do I understand correctly that your fw works on all LAN gateways?

**Comment by Koenkk** [comment]
Great, thanks!

**Comment by xyzroe** [comment]
Thanks!

**Comment by xyzroe** [comment]
You decided to make a video in the form of a hyperlink - but this leads to video downloading, not watching. Githab can make a beautiful built -in player, just by the link, but apparently the engine is so used for the site. Should I download it to some kind of hosting video to get a code for a built -in player? Or is it not worth it, what do you think?

**Pull request merged**